src/praisonai/praisonai/__main__.py

```
@@ -3,92 +3,98 @@
-        finally:
-            sys.argv = original
-        return
+            from praisonai.cli.app import app, register_commands
+            register_commands()
+
+            import typer.main
+            import click
+            click_app = typer.main.get_command(app)
+            ctx = click.Context(click_app, info_name="praisonai")
+            commands = set(click_app.list_commands(ctx))
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)